File: ed1/gara3/lollatz/att/lollatz.c
#include <stdio.h>
#include <assert.h>

int afaikdiyrotfl(int N) {
    // Mettete qui il codice della soluzione
    return 42;
}


int main() {
    FILE *fr, *fw;
    int N, i;

    fr = fopen("input.txt", "r");
    fw = fopen("output.txt", "w");
    assert(1 == fscanf(fr, "%d", &N));

    fprintf(fw, "%d\n", afaikdiyrotfl(N));
    fclose(fr);
    fclose(fw);
    return 0;
}

File: ed1/gara3/lollatz/att/lollatz.pas
function afaikdiyrotfl(N: longint): longint;
begin
    (* Mettete qui il codice della soluzione *)
    afaikdiyrotfl := 42;
end;


var
    N     : longint;
    fr, fw: text;

begin
    assign(fr, 'input.txt');
    assign(fw, 'output.txt');
    reset(fr);
    rewrite(fw);
    readln(fr, N);

    writeln(fw, afaikdiyrotfl(N));
    close(fr);
    close(fw);
end.
